### ISSUE TITLE
Support Multiple API Tokens

### ISSUE THREAD

**Issue by mkonda** [andjosh/clubhouse-cli#213]
### Expected Behavior
I use Clubhouse with several organizations.  I would like to be able to: 
1.   Ideally:  get integrated results
2.  Backup case:  have a way to specify a different config file for each org and 

### Actual Behavior
Appears to support one main config.

### Steps to Reproduce
club install (set up token 1)
club install (set up token 2)
There will be one config file.

```
mk@om crush % club install
A configuration/token is already saved. To override, re-run with --force
```

### Notes

I plan to try to implement this and offer a pull request - I just wanted to put it here in case either someone already knows this is an issue or has an approach.

**Comment by andjosh** [comment]
@mkonda You're correct, the current config and install process only allows for one token. 

But I think it's a good idea that can definitely be implemented!
